app/utils/Strings.scala

```diff
@@ -112,4 +112,51 @@ object Strings {
   def alpha(length: Int): String =
     Random.alphanumeric.filter(_.isLetter).take(length).mkString
 
+  def declarationGoodsItemNumber(): String =
+    // [1-9][0-9]{0,2}|[1][0-9]{3}
+    Seq(between1And9, between1And9).mkString
```

Comment:
second part of Seq should be between 0 to 9?

Reply:
Does it matter in this case? This is only generating a test payload. I'm content with what it is.

Reply:
yeah should be fine. Just in case if we wanted to follow the regex with content.